tests/fixture_setup/_fixtures.py

```diff
@@ -339,6 +344,12 @@ class FakeStoreSearchServerRunning(FakeServerRunning):
 class StagingStore(fixtures.Fixture):
     def setUp(self):
         super().setUp()
+        self.useFixture(
+            fixtures.EnvironmentVariable(
+                "SNAPU_STORE_DASHBOARD_ROOT_URL",
```

Comment:
oh, this needs fixing. This was in preparation of more than anything (as we only test commands from integration tests).